packages/editor-ui/src/components/RunData.vue

```diff
@@ -656,9 +657,9 @@ export default mixins(externalHooks, genericHelpers, nodeHelpers, pinData).exten
 				defaults.push({ label: this.$locale.baseText('runData.binary'), value: 'binary' });
 			}
 
-			if (this.isPaneTypeInput) {
-				defaults.unshift({ label: this.$locale.baseText('runData.schema'), value: 'schema' });
-			}
+			// For the input panel we want to show the schema view first
+			const addOperation = this.isPaneTypeInput ? 'unshift' : 'push';
+			defaults[addOperation]({ label: this.$locale.baseText('runData.schema'), value: 'schema' });
```

Comment:
not the most readable code here.. if you need a comment to explain it, it could probably be improved.
`if (this.paneType === 'input') { defaults.push('') } else { }` would be clearer for example..

Reply:
Fair enough, adjusted. Have another look, pls